CMakeLists.txt

```diff
@@ -211,8 +211,13 @@ IF (CMAKE_SYSTEM_NAME STREQUAL Windows)
     SET(CPACK_NSIS_HELP_LINK "https://vega-strike.org")
     SET(CPACK_NSIS_URL_INFO_ABOUT "https://vega-strike.org/about")
     SET(CPACK_NSIS_CONTACT "maintainers@vega-strike.org")
-    SET(CPACK_NSIS_MODIFY_PATH ON)
-
+    SET(CPACK_NSIS_MODIFY_PATH OFF)
+    SET(CPACK_NSIS_CREATE_ICONS_EXTRA
+        "CreateShortCut '$SMPROGRAMS\\\\$STARTMENU_FOLDER\\\\Vega Strike Upon the Coldest Sea.lnk' 'C:\\\\Program Files\\\\VegaStrike-0.9\\\\bin\\\\vegastrike-engine.exe' -D\"$INSTDIR\\share\\vegastrike\""
```

Comment:
could we use the variables from L26-L28 to build the path here?

Also, we do we need to use `\`? Windows will typically use `/` just fine too, and would make it cleaner:

```
"CreateShortCut '$SMPROGRAMS/$STARTMENU_FOLDER/Vega Strike Upon the Coldest Sea.lnk' 'C:/Program Files/VegaStrike-0.9/bin/vegastrike-engine.exe' -D\"$INSTDIR/share/vegastrike\""
```

Reply:
Yeah, the forward slashes didn't work. Backslashes it is.

Reply:
Supposedly it works if you use it within a powershell script. However, I'm not convinced it's worth the effort. Are we even requiring powershell on windows for Vegastrike?

```
$shell = New-Object -comObject WScript.Shell
$shortcut = $shell.CreateShortcut($shortcutPath)
$shortcut.TargetPath = $targetPath
$shortcut.Arguments = ""  # You can add arguments here if needed
$shortcut.Save()
```

Reply:
We are not requiring PowerShell on Windows for Vega Strike. That's why I didn't want to use anything PowerShell-specific.

Reply:
@royfalk I've done in command.com and cmd.exe for years. But not going to force it.

@stephengtuggy thanks for tryiing